test/strategies/kandel/abstract/CoreKandel.gas.t.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.10;
+
+import "./KandelTest.t.sol";
+import {TestToken} from "mgv_test/lib/tokens/TestToken.sol";
+import {Kandel} from "mgv_src/strategies/offer_maker/market_making/kandel/Kandel.sol";
+import {PinnedPolygonFork} from "mgv_test/lib/forks/Polygon.sol";
+
+abstract contract CoreKandelGasTest is KandelTest {
+  uint internal completeFill_;
+  uint internal partialFill_;
+
+  function setUp() public virtual override {
+    super.setUp();
+    vm.prank(maker);
+    kdl.setCompoundRates(10 ** PRECISION, 10 ** PRECISION);
+  }
+
+  function __deployKandel__(address deployer, address) internal virtual override returns (GeometricKandel kdl_) {
```

Comment:
Since we deploy a Kandel, is this is NoRouterKandelGasTest? and not CoreKandelGasTest?